src/edu/csus/ecs/pc2/core/imports/clics/CLICSJsonUtilities.java

```diff
@@ -110,22 +117,30 @@ public static void addGroupWinners(IInternalContest contest, Run[] runs, List<CL
             return;
         }
 
+        
         /**
          * Group Id and Client/Team Id
          */
         Map<Group, ClientId> groupWinners = new HashMap<Group, ClientId>();
+        try {
```

Comment:
Perhaps add a comment describing what this block of code is supposed to be doing.

Reply:
Pushed updated commnts about groupWinners map and how group winners are 
added to the map.